src/SharedLibrary/GeneratedCode/SharedLibrary_HashOneHotVectorizerFeaturizer.cpp

```diff
@@ -12,6 +12,7 @@
 
 // These method(s) are defined in SharedLibrary_Common.cpp
 ErrorInfoHandle * CreateErrorInfo(std::exception const &ex);
+std::chrono::system_clock::time_point CreateDateTime(DateTimeParameter const &param);

```

Comment:
Is this going to be in all the `SharedLibrary` code even if they don't use DateTIme stuff?

Reply:
Yes, however the linker will remove the code if it isn't being used.